app-misc/dateutils/dateutils-0.4.7.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="command line date and time utilities"
+HOMEPAGE="https://www.fresse.org/dateutils/"
+
+case "${PV}" in
+	9999)
+		inherit git-r3 autotools
+		EGIT_REPO_URI="https://github.com/hroptatyr/dateutils.git"
+		;;
+	*)
+		SRC_URI="https://bitbucket.org/hroptatyr/dateutils/downloads/${P}.tar.xz"
```

Comment:
We're nearly there, promise! We're using both github and bitbucket. Any reason for that?

Please add whichever is the official one / primary to HOMEPAGE too (after the existing URL).

Reply:
On their homepage it names github as primary host and bitbucket for downloads. It is probably more logical to use github for both EGIT_REPO_URI and SRC_URI, I'll change it.

Reply:
Nevermind, their tarballs on bitbucket are different from the tarballs on github. The bitbucket tarball contains:
+ aclocal.m4
+ configure (shell script)
+ Makefile.in
+ version.mk

Reply:
I was looking at the github tags instead of the github releases, the release tarballs are the same on github and bitbucket. I will change the SRC_URI to use the github tarballs.